src/environments/environment.prod.ts

```diff
@@ -1,4 +1,4 @@
 export const environment = {
   production: true,
-  apiUrl: "https://dam14n.github.io/miaprova-test/"
+  apiUrl: "http://localhost:3000"
```

Comment:
Same as in this MR https://github.com/Dam14n/miaprova-test/pull/17/files#r782223517